ruby_on_rails_example/config/credentials.yml.enc

```diff
@@ -0,0 +1 @@
+G0kCcett3zVytSazm6tWwEcW+9e6GrxXB3buqq4p8yZWBawNL8hUWwgpXgjvSoOACA7qtRyf9oO8csBpsDE0+lpSv0eq6VFHWf0ioQf9qCUkO21U39llaByOTuUVjefM2XSQekqTfNez5l1J/0semYYq8ACv1fuHGxEoMFmDm8HMikp4bS1WYVueTmFVEbToXsQ/YXCMYGDef3qqhu/lM8z61QPwpbfk6KijrViciQY84Iq+Ie+EP9jUMI9lFlE8/tC8k6U0qki8XOpSUUJWmlP+w5zRibFWkRaiTlrbPZH1dTxOBF+41JUV5okDaBnNXjPwAMfvjLmXiKG+DKoWSy6yMXa/1HERIBYdnPhW2wkutXI/t1gm7CM44PcNzVLtsWYxu6CNEVzwQpHWWV2kDVOD+vApcg4E9vJT--LCrE17cXJmEKmy1O--3YvFqvjjAneUKCun08xQtQ==
```

Comment:
Sorry I know nothing of ROR - I take it this is ok to have checked in

Reply:
Good spot. It was generated by the 'rails new' command, not used for anything important. Probably good to get rid of in case one of the internal pipelines flags it.